src/makeCalculator.js

```diff
@@ -37,7 +37,50 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
```

Comment:
It would be safer to check if the callback is a function to avoid a potential TypeError. You could add a condition like: 'if (typeof callback !== 'function') { throw new Error('Callback must be a function'); }'.